### DISCUSSION TITLE
SSD201 and displays

### DISCUSSION THREAD

**Discussion by suarezvictor** [linux-chenxing/linux-chenxing.org#18] [General]
Hi everyone, I've got recently insterested on MStar chips

I know according to @fifteenhex that display control wasn't achieved yet
It seems that there are leaked SDK headers and .a binaries that may be useful to control a display, see github project referenced  here [http://dev.comake.online/forum.php?mod=viewthread&tid=54&extra=page%3D1](http://dev.comake.online/forum.php?mod=viewthread&tid=54&extra=page%3D1)

People at widora are also getting display output with the new SSD210 chip (QFN68), see[widora forums](https://sns.widora.io/topic/767/ssd210-demo%E6%9D%BF-%E4%B8%8B%E4%B8%80%E6%AD%A5%E5%87%86%E5%A4%87%E7%82%B9%E5%B1%8F/8)
I asked on that forum about the means to get it working.

Additionally, FYI, the guys from widora are planning a module pretty similar yo Industio's one as I've seen at the Telegram forum,
https://t.me/widora

Do you have another info in regards to display output? As far as I'm collecting useful information I'll post it here

**Comment by fifteenhex** [comment]
I've added two links into the u-boot source here:

https://github.com/linux-chenxing/linux-chenxing.org/blob/master/ip/index.md#display-pipeline

From what I can tell their u-boot supports a bootlogo feature that allows putting a logo on the screen before Linux starts.
There should be enough info there to get a basic framebuffer working.

If you want I can buy you a SSD202D board with display so you can work on this?
If we can get it working for the SSD202D then getting it working for the SSD210 should be a piece of cake as SigmaStar just reuse the same blocks in different combinations for their chips.
